src/js/models/covid19/DefCodesModel.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * DefCodesModel.js
+ * Created by Jonathan Hill 06/19/20
+ */
+
+const defCodesModel = {
```

Comment:
I would recommend that we should not move covid19 to the v2 folder. I noticed that someone did that which I do not agree with.  V2 folder should be solely for V2 data or components or whatever.  Covid 19 is not a version 2 of anything.

Reply:
v2 means version 2 of our data models (not a feature) - using prototypal inheritance instead of Immutable records used in old data models. Agree that it can be confusing and we may ultimately want to move the old models into a v1 folder instead